scripts/check-broken-links.mjs

```diff
@@ -3,17 +3,60 @@ import { LinkChecker } from 'linkinator'
 import chalk from 'chalk'
 import {
   getSitemapLocations,
-  splitPagesIntoSegments,
+  splitPagesIntoBatches,
+  getPagesSlice,
 } from '../playwright/utils/getSitemapLocations.js'
 import fs from 'fs'
 
 const OPTIONS = {
   sitemapUrl: process.env.SITE_URL || 'http://www.va.gov/',
-  batchSize: process.env.BATCH_SIZE || 32,
+  // totalInstances is the number of instances to split work among.
+  totalInstances: process.env.TOTAL_INSTANCES || 1,
+  // instanceNumber is the specifc instance currently running the check.
+  // In this and totalInstances case, the defaults will check the entire set.
+  instanceNumber: process.env.INSTANCE_NUMBER || 1,
+  // batchSize is the number of pararllel link check processes to run.
+  batchSize: process.env.BATCH_SIZE || 20,
```

Comment:
This can be played with. I had better luck with 20 over 32. This was not verified scientifically, but I suspect that each runner has a limit on outgoing network connections.